charts/ingress-nginx/values.yaml

```diff
@@ -251,20 +251,37 @@ controller:
   ## Liveness and readiness probe values
   ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
   ##
+  # startupProbe:
+  #   httpGet:
+  #     path: "/healthz" # should match container.healthCheckPath
+  #     port: 10254
+  #     scheme: HTTP
+  #   initialDelaySeconds: 5
+  #   periodSeconds: 5
+  #   timeoutSeconds: 2
+  #   successThreshold: 1
+  #   failureThreshold: 5
   livenessProbe:
-    failureThreshold: 5
+    httpGet:
+      path: "/healthz" # should match container.healthCheckPath
```

Comment:
the lint is failing because the comment needs to be in the top and not in the side of the variable

```
# should match container.healthCheckPath
path: "/healthz"
```

Reply:
I made the requested changes